src/Microsoft.AspNet.Mvc.ViewFeatures/Resources.resx

```diff
@@ -259,4 +259,7 @@
   <data name="RemoteAttribute_RemoteValidationFailed" xml:space="preserve">
     <value>'{0}' is invalid.</value>
   </data>
+  <data name="HtmlGenerator_FieldNameCannotBeNullOrEmpty" xml:space="preserve">
+    <value>The field name for Html field cannot be Null or Empty. Please provide a non-Empty field name OR use 'EditorFor' as a workaround with parameter name=null.</value>
```

Comment:
@dougbu Thanks! 
There is no method `EditorFor` in `IHtmlHelper`. There is `EditorForModel`. Did you mean that? Also `EditorForModel` is an extension method so, `nameof` doesn't work with it. So I will have to hard code the name if that's the method you intend here.